CMakeLists.txt

```diff
@@ -70,7 +70,7 @@ add_compile_definitions(
 add_compile_options(/diagnostics:caret /W4 /WX /w14265 /w15038 /d1FastFail /guard:cf /Z7 /Gm- /Gy /Zp8 /std:c++latest /permissive- /Zc:threadSafeInit- /Zl /ZH:SHA_256)
 
 set(VCLIBS_DEBUG_OPTIONS "/Od")
-set(VCLIBS_RELEASE_OPTIONS "/O2;/Os") # TRANSITION: Potentially remove /Os
+set(VCLIBS_RELEASE_OPTIONS "/O3")

```

Comment:
One does not simply remove `/Os` #2108

Reply:
Also, MSVC has no `/O3` option (https://docs.microsoft.com/en-us/cpp/build/reference/compiler-options-listed-alphabetically?view=msvc-160).

Reply:
Pretty sure this is made with LLVM? either that or something is odd

Reply:
This repository is MSVC STL.
So it is built by MSVC.
Although the resulting binaries along with headers can be consumed by clang as well.

Reply:
> So it is built by MSVC.
> Although the resulting binaries along with headers can be consumed by clang as well.

That's correct. Although we support Clang/LLVM as a first-class citizen for users building their own code, we have never attempted to build the STL's DLLs and LIBs with Clang/LLVM. This is partially for technical reasons (some translation units in the DLL/LIBs are built with options that Clang/LLVM doesn't implement, like `/EHa` and `/clr:pure`), partially for complexity (every additional "mode" incurs development and testing costs), and mostly for lack of benefit (we would need to see significant speed or size improvements which seem unlikely, given what code is in the DLL/LIB; notably `<charconv>`'s logic is header-only so it benefits when users use Clang/LLVM).